templates/compose/opnform.yaml

```
@@ -0,0 +1,226 @@
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 45s
+
+  postgresql:
+    image: postgres:16
+    volumes:
+      - opnform-postgresql-data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: ${SERVICE_USER_POSTGRESQL}
+      POSTGRES_PASSWORD: ${SERVICE_PASSWORD_POSTGRESQL}
+      POSTGRES_DB: ${POSTGRESQL_DATABASE:-opnform}
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U $${POSTGRES_USER} -d $${POSTGRES_DB}"]
+      interval: 5s
+      timeout: 20s
+      retries: 10
+
+  redis:
+    image: redis:7
+    environment:
+      REDIS_PASSWORD: ${SERVICE_PASSWORD_64_REDIS}
+    volumes:
+      - redis-data:/data
+    command: ["redis-server", "--requirepass", "${SERVICE_PASSWORD_64_REDIS}"]
+    healthcheck:
+      test: ["CMD", "redis-cli", "-a", "${SERVICE_PASSWORD_64_REDIS}", "PING"]
+      interval: 10s
+      timeout: 30s
+      retries: 3
+
+  # The nginx reverse proxy.
+  # used for reverse proxying the API service and Web service.
+  nginx:
+    image: nginx:latest
+    volumes:
+      - type: bind
+        source: ./nginx/nginx.conf.template
+        target: /etc/nginx/conf.d/opnform.conf
+        read_only: true
+        content: |
+          map $request_uri $api_uri {
+              ~^/api(/.*$) $1;
+              default $request_uri;
+          }
+
+          server {
+              listen 80 default_server;
+              root /usr/share/nginx/html/public;
+
+              access_log /dev/stdout;
+              error_log /dev/stderr error;
+
+              index index.html index.htm index.php;
+
+              location / {
+                  proxy_http_version 1.1;
+                  proxy_pass http://opnform-ui:3000;
+                  proxy_set_header X-Real-IP $remote_addr;
+                  proxy_set_header X-Forwarded-Host $host;
+                  proxy_set_header X-Forwarded-Port $server_port;
+                  proxy_set_header Upgrade $http_upgrade;
+                  proxy_set_header Connection "Upgrade";
+              }
+
+              location ~/(api|open|local\/temp|forms\/assets)/ {
+                  set $original_uri $uri;
+                  try_files $uri $uri/ /index.php$is_args$args;
+              }
+
+              location ~ \.php$ {
+                  fastcgi_split_path_info ^(.+\.php)(/.+)$;
+                  fastcgi_pass opnform-api:9000;
+                  fastcgi_index index.php;
+                  include fastcgi_params;
+                  fastcgi_param SCRIPT_FILENAME $document_root/index.php;
+                  fastcgi_param REQUEST_URI $api_uri;
+              }
+
+              # Deny access to . files
+              location ~ /\. {
+                  deny all;
+              }
+          }
+    environment:
+      - SERVICE_FQDN_NGINX
+    depends_on:
+      - opnform-api
+      - opnform-ui
+    healthcheck:
+      test: ["CMD", "nginx", "-t"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 40s
```

Volumes section were missing
```suggestion
      start_period: 40s
volumes:
  redis-data:
  opnform-postgresql-data:
  api-storage:
```

---

This was a suggested change https://github.com/coollabsio/coolify/pull/5875/files/eef0e940e704f5413b309cc1d0099b26cd7267ee#r2104770506